src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -375,13 +361,11 @@ private void createGlobalThroughWidget()
         clearGlobalThrough.setText("clear");
         clearGlobalThrough.addSelectionListener(new SelectionListener()
         {
-            @Override
             public void widgetSelected(SelectionEvent e)
             {
                 backend.clearGlobalThrough();
             }
 
-            @Override
             public void widgetDefaultSelected(SelectionEvent e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)